lips/lip-6.mdx

```diff
@@ -10,27 +10,26 @@ created: 07/30/2020
-The Libra Payment Network (“LPN”) is a blockchain-backed payment platform and an infrastructure for stablecoins. Libra Networks (“Libra Networks” or “LN”) is a whole subsidiary of the Libra Association (the “Association” or “LA”) which is in the process of applying for an LPN payment system license from the Swiss Financial Market Supervisory Authority (“FINMA”). Once granted, LN will be responsible for ensuring that LPN operates in compliance with the payment system license from FINMA (the "FINMA License").
+The Libra Payment Network ("LPN") and the Libra Association ("LA") manage the LibraRoot account. The LPN is a blockchain-backed payment platform and an infrastructure for stablecoins. Libra Networks (or “LN”) is a whole subsidiary of the LA. The LA is in the process of applying for an LPN payment system license from the Swiss Financial Market Supervisory Authority (“FINMA”). Once granted, LN will be responsible for ensuring that LPN operates in compliance with the payment system license from FINMA (the "FINMA License").
 
-The Libra Association is an independent membership organization that is responsible for the LPN technology and the development of the Libra project. The Association is governed by the Association Council, which is comprised of one representative per Association member. The Association Council manages the technology roadmap of the platform and oversees and maintains the Libra Improvement Proposal and Libra Upgrade Process, often through subcommittees of the Council such as the Technical Steering Committee (the “TSC”) and the Association Board.
+The LA is an independent membership organization that is responsible for the LPN technology and the development of the Libra project. The LA is governed by the Association Council, which is comprised of one representative per Association member. The Association Council manages the technology roadmap of the platform and oversees and maintains the Libra Improvement Proposal and Libra Upgrade Process, often through subcommittees of the Association Council such as the Technical Steering Committee (the “TSC”) and the Association Board.
 
-The Association together with LPN, through Association Operation Services (the "AOS"), jointly manage the LibraRoot account that determines the composition of the Validator Set at any given time. Reasons for changing the Validator Set include new members joining the Association or existing members leaving it and actions needed to comply with LN’s obligations under its FINMA License designed to guarantee the secure and performant operation of the platform.
+The LA together with LPN jointly manage the LibraRoot account that determines the composition of the Validator Set at any given time. Reasons for changing the Validator Set include new members joining the Association or existing members leaving it and actions required to comply with LN’s obligations under its FINMA License designed to guarantee the secure and performant operation of the platform.
```

Comment:
LPN -> LN

Reply:
just say "required for compliance" and omit the rest. There are many rules and licenses, as well as own risk assessment, that may bring LN to decide to take a governance action